### ISSUE TITLE
www.snapdrop.net = SSL_ERROR_INTERNAL_ERROR_ALERT

### ISSUE THREAD

**Issue by CouldBeThis** [RobinLinus/snapdrop#341]
If you go to https://www.snapdrop.net/ instead of https://snapdrop.net/ there is an error

> 
> Secure Connection Failed
> 
> An error occurred during a connection to www.snapdrop.net. Peer reports it experienced an internal error.
> 
> Error code: SSL_ERROR_INTERNAL_ERROR_ALERT
> 
>     The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
>     Please contact the website owners to inform them of this problem.
> 
> Learn more…

**Comment by Bellisario** [comment]
Yes, @CouldBeThis, some sites are not enabled for www, in fact, normally, www is not anymore used all the time (you don't need, usually, put www all the times) but only managed and chosen by the browser. I don't know if this is a bug, or a chosen behavior by @RobinLinus.
